### PR TITLE
hometask done

### PR DESCRIPTION
**Pull request by sergey87chuvasov**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1 +1,82 @@
...
+
+test = Boolean(test);
+console.log(typeof test);
+
+// Задача 1
+/* Дан массив [2, 3, 5, 8]. 
+С помощью цикла for найдите произведение элементов этого массива. 
+Результат выведите в консоль. */
+
+let arr  = [2, 3, 5, 8];  // индекс массива 0,1,2,3; длинна массива = 4;
```
Comment (ghost) [inline js/main.js]: Should be a `const`.

File: js/main.js [hunk 2]
```diff
@@ -1 +1,82 @@
...
+for (let i = 0; i < arr2.length; i++) {
+    if (arr2[i] > 5 && arr2[i] < 10) {  // применили логический оператор И &&
+        console.log('Вывести число:', arr2[i]) // 8,6
+    }
+}
+
+/*  Задача 3 - Дан массив [2, 5, 8, 15, 0, 6, 20, 3]. 
+С помощью цикла for и оператора if выведите в консоль четные элементы массива. 
+0 - это четное число - но тут убрал его для вывода */
+let arr3 = [2, 5, 8, 15, 0, 6, 20, 3];
```
Comment (ghost) [inline js/main.js]: Should be a `const`.

File: js/main.js [hunk 3]
```diff
@@ -1 +1,82 @@
...
+}
+console.log(multipl1); //240
+
+// Задача 2
+/*
+Дан массив [2, 5, 8, 15, 0, 6, 20, 3]. 
+С помощью цикла for и оператора if выведите в консоль те элементы массива, 
+которые больше 5-ти, но меньше 10-ти. */
+
+let arr2 = [2, 5, 8, 15, 0, 6, 20, 3];
```
Comment (ghost) [inline js/main.js]: Should be a `const`.

File: js/main.js [hunk 4]
```diff
@@ -1 +1,82 @@
...
+        console.log('Вывести число:', arr2[i]) // 8,6
+    }
+}
+
+/*  Задача 3 - Дан массив [2, 5, 8, 15, 0, 6, 20, 3]. 
+С помощью цикла for и оператора if выведите в консоль четные элементы массива. 
+0 - это четное число - но тут убрал его для вывода */
+let arr3 = [2, 5, 8, 15, 0, 6, 20, 3];
+for (let i = 0; i < arr3.length; i++) {
+    if (arr3[i] % 2 === 0 && arr3[i] !== 0) {
```
Comment (ghost) [inline js/main.js]:
```suggestion
if(!(arr3[i] % 2)){}
```

File: js/main.js (hunk 1 above)
Comment (sergey87chuvasov) [reply js/main.js]: fixed mistake

File: js/main.js (hunk 2 above)
Comment (sergey87chuvasov) [reply js/main.js]: fixed mistake

File: js/main.js (hunk 3 above)
Comment (sergey87chuvasov) [reply js/main.js]: fixed mistake

File: js/main.js (hunk 4 above)
Comment (sergey87chuvasov) [reply js/main.js]: fixed mistake

**Pull request merged**